packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -737,18 +736,17 @@ export default class Meetings extends WebexPlugin {
   }
 
   /**
-   * Explicitly sets up the meetings plugin by registering
-   * the device, connecting to mercury, and listening for locus events.
+   * Do some pre-initialization before the full registration is done.
    *
    * @returns {Promise}
    * @public
    * @memberof Meetings
    */
-  public register() {
+  public preregister(): Promise<any> {
```

Comment:
splitting register into preregister + register so that part can be called before the embedded interstitial and part can be called after